sdk/akari_client/setup.py

```diff
@@ -36,7 +36,7 @@
         "depthai": [
             "matplotlib==3.6.2",
             "depthai==2.19.1.0",
-            "opencv-python-headless==4.6.0.66",
+            "opencv-python==4.7.0.72",
```

Comment:
```suggestion
            "opencv-python==4.6.0.66",
```

ここバージョンアップしているのはなぜでしょう？

Reply:
もし可能であればこのPRで opencv-python の強い version 制約も除きたいです。いかがでしょう？

Reply:
バージョンは動確時のものに合わせてしまいました。
バージョン制約抜いてしまいますね。

Reply:
ありがとうございます！

> バージョン制約抜いてしまいますね

https://pypi.org/project/opencv-python/#history から semantic versioninig に則っていることを仮定してそれっぽいものにするのが良いのかなと思ってました（例えば `>4.5,<5.0` とか？）が、特に動作確認できているわけではないのでバージョン制約そのものを抜くのでも良い気がしますね。SGTMです。